src/test/java/org/opentripplanner/middleware/OtpDispatcherHelper.java

```diff
@@ -0,0 +1,40 @@
+package org.opentripplanner.middleware;
+
+import org.opentripplanner.middleware.otp.OtpDispatcherImpl;
+import org.opentripplanner.middleware.otp.OtpDispatcherResponse;
+import org.opentripplanner.middleware.otp.response.Response;
+import org.opentripplanner.middleware.utils.JsonUtils;
+
+/**
+ * Used to provide real plan responses from OTP
+ */
+public class OtpDispatcherHelper {
+
+    private static final String OTP_SERVER = "https://fdot-otp-server.ibi-transit.com";
+    private static final String OTP_SERVER_PLAN_END_POINT = "/otp/routers/default/plan";
```

Comment:
These should probably be obtained from application config.

Reply:
Updated